app/code/Magento/Wishlist/Test/Mftf/Test/StorefrontDisabledCustomerWishlistFunctionalityTest.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontDisabledCustomerWishlistFunctionalityTest">
+        <annotations>
+            <features value="Wishlist"/>
+            <stories value="Disabled Wishlist Functionality"/>
+            <title value="Wishlist Functionality is disabled in system configurations and not visible on FE"/>
+            <description value="Customer should not see wishlist functionality if it's disabled"/>
+            <group value="wishlist"/>
+            <group value="configuration"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set wishlist/general/active 0" stepKey="disableWishlist"/>
+            <magentoCLI command="cache:clean" stepKey="cleanCache"/>
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="SimpleProduct" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
```

Comment:
After creating products and categories you should run cron for index group, as tests might be run in "on schedule" indexing mode

Reply:
Added Cron Magento CLI command on 27 line
Please check.
Thanks!